scripts/unix_share_build.sh

```diff
@@ -289,6 +291,11 @@ function mac_build_plugin_pkg(){
     cp /usr/local/opt/libaec/lib/libsz.dylib .
     install_name_tool -change /usr/local/opt/libaec/lib/libsz.dylib @rpath/libsz.dylib libsvalinn_plugin.so
 
+    if [ "$1" == "2022.4" ] ; then
+    cp /usr/local/opt/libaec/lib/libsz.2.dylib .
+    install_name_tool -change /usr/local/opt/libaec/lib/libsz.2.dylib @rpath/libsz.2.dylib libsvalinn_plugin.so
+    fi
```

Comment:
What if we made the lib name or version a variable based on the Cubit version, and then use that in various lines that are common (does that make sense?)

Reply:
Do you mean a $cubit_version variable instead for using an argument ?

Reply:
Here's what I was thinking:
```suggestion
    libsz=libsz.dylib
    if [ "$1" == "2022.4" ] ; then
    libsz=libsz.2.dylib
    fi    

        cp /usr/local/opt/libaec/lib/$libsz .
    install_name_tool -change /usr/local/opt/libaec/lib/$libsz @rpath/$libsz libsvalinn_plugin.so
```